client/components/Project/ProjectDetail.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import ProjectActionCreator from '../../actions/project'
+import PropTypes from 'prop-types'
+import {Link} from 'react-router-dom'
+import Card, {CardContent} from 'material-ui/Card'
+import Typography from 'material-ui/Typography'
+import styles from './ProjectDetail.scss'
+
+class ProjectDetail extends Component {
+  constructor (args) {
+    super(args)
+    this.state = {projectId: this.props.match.params.id}
```

Comment:
if you are using this.state.projectId only in componentDidMount, use this.props.match.params.id directly in there. No need to use state.

Reply:
Thanks for your suggestion!
I had revised the two files according to your request. 
Thanks again!

Reply:
Actually they belong to two different stories: the first one is #156253778, and the second one is #154707979, I revised both file in the first one by accident, should I revised them in their own branches? Besides, /profile is not working, I can't see the project list when click profile

Reply:
But the projectId is being used anywhere else? I can't find any reference to it unless in the componenDidMount method.